src/components/Categories/LoanSpotlight.vue

```diff
@@ -0,0 +1,255 @@
+				) {
+					values {
+						id
+						description
+						lenderRepaymentTerm
+						anonymizationLevel
+						geocode {
+							city
+							country {
+								name
+							}
+						}
+						image {
+							id
+							default: url(customSize: $imgDefaultSize)
+							retina: url(customSize: $imgRetinaSize)
+						}
+					}
+				}
+			}
+		}
+	}
+`;
+
+function getTargetedChannel(targetedRoute, allChannels) {
+	const loanChannels = allChannels.lend?.loanChannels?.values;
+	const targetedLoanChannel = loanChannels.filter(
+		loanChannel => loanChannel.url.split('/').pop() === targetedRoute
+	);
+	// no channel that matches the name recommended-by-lenders
+	if (targetedLoanChannel.length === 0) {
+		// return id for channel women
+		return 5;
+	}
+	// recommended-by-lenders channel exists but no loans exist within it
+	if (targetedLoanChannel.length !== 0 && targetedLoanChannel[0].loans.totalCount === 0) {
+		return 5;
+	}
+	// isolate recommended-by-lenders channel id
+	return targetedLoanChannel[0]?.id || null;
+}
+
+function filterByAnonymizationLevel(spotlightData) {
+	const firstFiveRecommendedLoans = spotlightData.lend?.loanChannelsById[0]?.loans?.values ?? [];
+	const nonAnonymousLoans = firstFiveRecommendedLoans.filter(
+		loan => loan.anonymizationLevel !== 'full'
+	);
+	return nonAnonymousLoans[0] || {};
+}
+
+export default {
+	name: 'LoanSpotlight',
+	components: {
+		KvGrid,
+		KvButton,
+		KvPageContainer,
+		KvResponsiveImage
+	},
+	inject: ['apollo', 'cookieStore'],
+	data() {
+		return {
+			spotlightPlaceholderImageCTF: '',
+			spotlightLoan: {},
+			targetedLoanChannelID: null
+		};
+	},
+	computed: {
+		altText() {
+			return "Today's loan spotlight";
+		},
+		getSpotlightText() {
+			return toParagraphs(this.spotlightLoan.description);
+		},
+		getSpotlightLoanLocation() {
+			if (this.spotlightLoan.geocode.city && this.spotlightLoan.geocode.country.name) {
+				return `${this.spotlightLoan.geocode.city}, ${this.spotlightLoan.geocode.country.name}`;
+			}
+			return `${this.spotlightLoan.geocode?.country?.name ?? ''}`;
+		},
+		imageExists() {
+			return this.spotlightLoan.image?.default ?? '';
+		},
+		getSpotlightImage() {
+			if (this.imageExists === '') {
+				return [['small', this.spotlightPlaceholderImageCTF]];
+			}
+			return [['small', this.spotlightLoan.image.retina], ['small retina', this.spotlightLoan.image.retina]];
+		}
+	},
+	apollo: {
+		preFetch(config, client) {
+			return client.query({
+				query: allChannelsQuery
+			}).then(({ data }) => {
+				const targetedLoanChannelURL = routePath;
+				const targetedLoanChannelID = getTargetedChannel(targetedLoanChannelURL, data);
+				return client.query({
+					query: spotlightLoanQuery,
+					variables: {
+						ids: [targetedLoanChannelID],
+					},
+				});
+			});
+		},
+	},
+	created() {
+		let allChannelsData = {};
+		try {
+			allChannelsData = this.apollo.readQuery({
+				query: allChannelsQuery,
+			});
+		} catch (e) {
+			logReadQueryError(e);
```

Comment:
We generally also pass in the location of the error as the second arg:
`logReadQueryError(e, 'LoanSpotlight created');`